File: app/src/main/java/com/tiny/demo/firstlinecode/rxjava2/blog2/entity/Person.java
package com.tiny.demo.firstlinecode.rxjava2.blog2.entity;

import java.util.ArrayList;
import java.util.List;

/**
 * Desc:
 *
 * @author tiny
 * @date 2018/6/7 上午12:53
 */
public class Person {
    private String name;
    private List<Plan> planList = new ArrayList<>();

    public Person(String name, List<Plan> planList) {
        this.name = name;
        this.planList = planList;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Plan> getPlanList() {
        return planList;
    }

    public void setPlanList(List<Plan> planList) {
        this.planList = planList;
    }
}

File: app/src/main/java/com/tiny/demo/firstlinecode/kotlin/primer/project06/Person.kt
package com.tiny.demo.firstlinecode.kotlin.primer.project06

/**
 * desc 构造函数
 *
 * @author wangjianzhou@qding.me
 * @version version
 * @date 2018/8/13 4:13 PM
 */
class Person {
    init {
        println(javaClass.simpleName + " inited")
    }

//    constructor() {
//        println(javaClass.simpleName + " 111")
//    }

    constructor(name: String) : super() {
        println(javaClass.simpleName + " 222")
    }
}